game/system/newpad.cpp

```diff
@@ -107,64 +139,93 @@ int IsPressed(MappingInfo& mapping, Button button, int pad = 0) {
   if (g_gamepad_buttons[pad][(int)button]) {
     return 1;
   }
-  auto key = mapping.pad_mapping[pad][(int)button];
+
+  int key = mapping.keyboard_button_mapping[pad][(int)button];
   if (key == -1)
     return 0;
   auto& keymap = mapping.buffer_mode ? g_buffered_key_status : g_key_status;
   ASSERT(key < NUM_KEYS);
   return keymap[key];
 }
 
+void SetAnalogAxisValue(int axis, double value) {
+  if (axis == GlfwKeyCustomAxis::CURSOR_X_AXIS || axis == GlfwKeyCustomAxis::CURSOR_Y_AXIS) {
+    value /= 18.0f;  // Arbituary value. Cursor delta generally a lot larger than scroll wheel delta
```

Comment:
Will the framerate have an effect on the cursor sensitivity? If the framerate is super high, the delta cursor positions will be smaller.

Reply:
Great catch! I only tested with 60 frames on my initial tests. Higher frame rate would reduce the delta cursor positions. I was thinking about making the delta configurable since different mouse(s) could have different DPIs. Would having a mouse sensitivity value in the json be a good idea?

Reply:
Yeah, that sounds like the best option.